ci-operator/step-registry/baremetalds/assisted/operator/upgrade/baremetalds-assisted-operator-upgrade-commands.sh

```diff
@@ -10,34 +10,13 @@ echo "************ baremetalds assisted operator upgrade command ************"
 # shellcheck source=/dev/null
 source "${SHARED_DIR}/packet-conf.sh"
 
-# # ZTP scripts have a lot of default values for the spoke cluster configuration. Adding this so that they can be changed.
-# if [[ -n "${ASSISTED_ZTP_CONFIG:-}" ]]; then
-#   readarray -t config <<< "${ASSISTED_ZTP_CONFIG}"
-#   for var in "${config[@]}"; do
-#     if [[ ! -z "${var}" ]]; then
-#       echo "export ${var}" >> "${SHARED_DIR}/assisted-ztp-config"
-#     fi
-#   done
-# fi
-
-# # Copy configuration for ZTP vars if present
-# if [[ -e "${SHARED_DIR}/assisted-ztp-config" ]]
-# then
-#   scp "${SSHOPTS[@]}" "${SHARED_DIR}/assisted-ztp-config" "root@${IP}:assisted-ztp-config"
-# fi
-
```

Comment:
Just taking a look now. `ASSISTED_ZTP_CONFIG` was created so that we wouldn't have to add every environment variable in the scripts into the step-registry like this. Removing it is extremely short-sighted the decision should be the same across all the ZTP scripts as well.

Since other scripts use this config all of the new ENV variables in this job should be added into the `ASSISTED_ZTP_CONFIG` instead of removing it.

Reply:
Thank you. I have updated the config accordingly https://github.com/openshift/release/pull/22240/files#diff-c57a0b6469c4c70a7423ed9ae84e626bc59f0347557a3b8356ea1247c7c31ec7R476-R489

Reply:
Some of the env vars need to be renamed in assisted-service. PR https://github.com/openshift/assisted-service/pull/2976